Makefile

```diff
@@ -18,6 +18,9 @@ test.e2e:
 test.e2e.single:
```

Comment:
can you also add the docker variant for single as well?

Reply:
ok